### PR TITLE
Change Kibana index patterns

### PR DESCRIPTION
**Pull request by monicasarbu**
Fixes PR https://github.com/elastic/beats-dashboards/pull/40 after being merged.

### CODE REVIEW

**Comment by monicasarbu** [comment]
@lmangani Here are couple of small fixes that I did after merging your PR in order. Thanks!

File: load.sh [hunk 1]
```diff
@@ -1,25 +1,89 @@
...
+        exit 1
+        ;;
+
+esac
+shift 2
+done
+
 DIR=dashboards
+echo "Loading dashboards to $ELASTICSEARCH in $KIBANA_INDEX using $CURL:"  
+exit 0
```
Comment (andrewkroh) [inline load.sh]: `exit` here?

File: load.sh [hunk 2]
```diff
@@ -1,25 +1,89 @@
...
+  Load the dashboards, visualizations and index patterns into the given Elasticsearch instance.
+
+Usage:
+
+  $0 -url http://localhost:9200 -user admin -index .kibana_env1
+
+Options:
+
+  -h | -help
+    Print the help menu.
```
Comment (andrewkroh) [inline load.sh:26]: Should there be a blank line here as there is between the other options?

Comment (monicasarbu) [inline load.sh:26]: yes, I noticed as well after I committed :)

File: load.sh (hunk 1 above)
Comment (monicasarbu) [inline load.sh]: Oh, good catch. I forgot to remove it. Thanks!

**Comment by andrewkroh** [comment]
LGTM

**Pull request merged**